### PR TITLE
[MU3] Removed revision h file

### PR DESCRIPTION
**Pull request by igorkorsukov**
Removed revision h file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,7 +27,6 @@ DocLib
 build.qt5
 *.sfd~
 /.project
-mscore/revision.h
```
Comment (Jojo-Schmitz) [inline .gitignore:30]: It might even stay here, for those that still use that hook?

**Comment by Jojo-Schmitz** [comment]
With this I guess we can finally also get rid if stull like thts in the release notes:`
'''
Important note!

Revision number (in mscore/revision.h) is incorrect in the assets automatically generated by GitHub. Use MuseScore-x.y.z.zip instead.
``

File: .gitignore (hunk 1 above)
Comment (igorkorsukov) [reply .gitignore]: returned

**Comment by Jojo-Schmitz** [comment]
The Mac build fail, `/bin/sh:  : command not found` seems to indicate some shell/environmane variable not expanding properly?

**Comment by igorkorsukov** [comment]
> The Mac build fail, `/bin/sh: : command not found` seems to indicate some shell/environment variable not expanding properly?
> 
> Maybe that `$(FORMATTER_START)`?

There was waste `\` after `$(FORMATTER_END);  `
now all right for MacOS, builded and packed right.

**Pull request merged**